spec/features/publish/courses/editing_course_fee_details_copy_content_spec.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+feature 'Editing fee details section, copying content from another course' do
+  scenario 'source course has fee details data to copy' do
+    given_i_am_authenticated_as_a_provider_user
+    and_there_is_a_course_i_want_to_edit
+    and_there_is_a_course_with_data_i_want_to_copy
+
+    when_i_visit_the_fee_details_edit_page
+    and_i_select_the_other_course_from_the_copy_content_dropdown
+
+    then_i_see_the_copied_course_data
+    and_i_see_the_warning_that_changes_are_not_saved
+    and_links_in_warning_match_input_ids
+  end
+
+  scenario 'copy course content options are available after validation' do
+    given_i_am_authenticated_as_a_provider_user
+    and_there_is_a_course_i_want_to_edit
+    and_there_is_a_course_with_data_i_want_to_copy
+
+    when_i_visit_the_fee_details_edit_page
+    when_i_submit_without_data
+    then_i_can_still_copy_content_from_another_course
+  end
+
+  private
+
+  def given_i_am_authenticated_as_a_provider_user
+    given_i_am_authenticated(user: create(:user, :with_provider))
+  end
+
+  def and_there_is_a_course_i_want_to_edit
+    given_a_course_exists(:secondary, additional_degree_subject_requirements: nil, degree_subject_requirements: nil)
+  end
+
+  def when_i_submit_without_data
+    fill_in 'Fees and financial support (optional)', with: 'a ' * 251
```

Comment:
If you are submitting without data, shouldn't it be:
```suggestion
    fill_in 'Fees and financial support (optional)', with: ''
```

Reply:
Ah good spot. The name of the method is wrong, this is an optional field so need to provide too many words to trigger validation.